lib/matplotlib/tests/test_axes.py

```diff
@@ -4335,6 +4337,50 @@ def test_pandas_indexing_hist():
     fig, axes = plt.subplots()
     axes.hist(ser_2)
 
+@image_comparison(baseline_images=['date_timezone_x'],
+                  extensions=['png'])
+def test_date_timezone_x():
+    #Tests issue 5575
+    time_index = [pytz.timezone('Canada/Eastern').localize(datetime.datetime(year=2016, month=2, day=22, hour=x)) for x in range(3)]
+
+    #Same Timezone
+    fig = plt.figure(figsize=(20,12))
+    plt.subplot(2, 1, 1)
+    plt.plot_date(time_index, [3]*3, tz='Canada/Eastern')
+
+    #Different Timezone
+    plt.subplot(2, 1, 2)
+    plt.plot_date(time_index, [3]*3, tz='UTC')
+
```

Comment:
Please use two empty lines between functions (this file is not yet checked against PEP8.)